create_coda_files/create_coda_files.py

```diff
@@ -1,55 +1,153 @@
+        'converts it to a CODAv2 file for verfication and manual coding')
+    parser.add_argument(
+        'user', help='User launching this program')
+    parser.add_argument(
+        'json_input_path', metavar='json-input-path',
+        help='Path to the input JSON file, containing a list of serialized'
+        'TracedData objects')
+    parser.add_argument(
+        'flow_name', metavar='flow-name',
+        help='Name of activation flow from which this data was derived')
+    parser.add_argument(
+        'variable_name', metavar='variable-name',
+        help='Name of message variable in flow')
+    parser.add_argument(
+        'traced_json_output_path', metavar='json-output-path',
+        help='Path to concatenated TraceData JSON')
+    parser.add_argument(
+        'coda_output_path', metavar='coda-output-path',
+        help='Path to a Coda file to write processed messages to')
+    parser.add_argument(
+        'coding_scheme_path', metavar='coding-scheme',
+        help='Path to coding scheme that is for this data')
+    parser.add_argument(
+        'is_yes_no', metavar='is-yes-no',
+        help='Is this variable a yes no question')
+    parser.add_argument(
+        'auto_cleaner', metavar='auto-cleaner',
+        help='automated cleaner to apply to this data')
 
     args = parser.parse_args()
     user = args.user
     json_input_path = args.json_input_path
     flow_name = args.flow_name
     variable_name = args.variable_name
+    traced_json_output_path = args.traced_json_output_path
     coda_output_path = args.coda_output_path
-    if not os.path.exists(prev_coda_path):
-        prev_coda_path = None
+    coding_scheme_path = args.coding_scheme_path
+    is_yes_no = args.is_yes_no
+    if args.auto_cleaner in AUTO_CLEANERS.keys():
+        auto_cleaner = args.auto_cleaner
+        auto_cleaner = AUTO_CLEANERS[auto_cleaner]
     else:
-        prev_coda_path = args.prev_coda_path
+        print('There is no matching cleaner for: {}'.format(args.auto_cleaner))
+        auto_cleaner = None
 
-    # Load data from JSON file
-    with open(json_input_path, "r") as f:
-        show_messages = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+    # Load in the coding scheme
+    code_scheme = _open_scheme(coding_scheme_path)
+
+    # Load traced data list from JSON file
+    with open(json_input_path, 'r') as f:
+        list_td = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
 
     # Filter out test messages sent by AVF.
-    show_messages = [td for td in show_messages if not td.get("test_run", False)]
+    list_td = [td for td in list_td 
+                if not td.get('test_run', False)]
+
+    # Keys used in the script
+    message_key = '{} (Text) - {}'.format(variable_name, flow_name)
+    coded_key = '{}_coded'.format(variable_name.lower())
+    id_field = '{}_id'.format(message_key)
+    time_key = '{} (Time) - {}'.format(variable_name, flow_name)
+    if is_yes_no == 'True':
+        is_yes_no = True
```

Comment:
Please parse your arg earlier - see comments on another PR

Reply:
Changed to only have two options True, False